frontend/src/App.svelte

```diff
@@ -1,17 +1,23 @@
 <script>
   import Interface from "./Interface.svelte";
 
-  export let title;
-  export let description;
-  export let theme;
-  export let dark;
-  export let input_components;
-  export let output_components;
-  export let examples;
-  export let fn;
-  export let root;
-  export let allow_flagging;
-  export let allow_interpretation;
+  export let title,
+    description,
+    article,
+    theme,
+    dark,
+    input_components,
+    output_components,
+    examples,
+    fn,
+    root,
+    space,
+    allow_flagging,
+    allow_interpretation,
+    live,
+    queue;
+
+  $: embedded = space && space.includes("/");
```

Comment:
Is this heuristic a little fragile? I can imaging cases where this could throw up false negatives and positives.